Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour {
+
+	public void StartGame () 
+	{
+		Debug.Log("STARTING GAME");
+		SceneManager.LoadScene (SceneManager.GetActiveScene ().buildIndex + 1);
+	}
+
+	public void QuitGame () 
+	{

```

Comment:
if you add the following property IF statement here, this logic will work in the editor 

```
  #if UNITY_EDITOR
         // Application.Quit() does not work in the editor so
         // UnityEditor.EditorApplication.isPlaying need to be set to false to end the game
         UnityEditor.EditorApplication.isPlaying = false;
 #else
         Application.Quit();
  #endif
```